lib/linguist/languages.yml

```diff
@@ -44,6 +44,16 @@ AGS Script:
   tm_scope: source.c++
   ace_mode: c_cpp
 
+Alex Haskell:
+  type: programming
+  group: Haskell
```

Comment:
Yes, [Alex](https://www.haskell.org/alex/) is a Haskell preprocessor (just like c2hs). As a preprocessor language it's similar to lex/flex, except that it uses/generates Haskell rather than C/C++.